base_search_mail_content/models/res_partner.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# © 2016 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# © 2016 Serpent Consulting Services Pvt. Ltd. (<http://www.serpentcs.com>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import models
+
+
+class Partner(models.Model):
+
+    _name = 'res.partner'
+    _inherit = ['res.partner', 'mail.thread']
```

Comment:
It already inherits or you wouldn't have chatter on partners, so this is not needed.